configure.sh

```diff
@@ -23,13 +22,17 @@ $(grep avx2 /proc/cpuinfo >/dev/null)
 AVX2=$?
 $(grep avx /proc/cpuinfo >/dev/null)
 AVX=$?
+$(grep mmx /proc/cpuinfo >/dev/null)
+MMX=$?
 
 if [ $AVX2 -eq 0 ]; then
   PLATFORM="haswell"
 elif [ $AVX -eq 0 ]; then
   PLATFORM="sandybridge"
+elif [ $AVX -eq 0]; then
+  PLATFORM="nehalem"
 else
-  echo "ERROR: STOKE is currently only supported on sandybridge or haswell machines.  You appear to have an older CPU."
+  echo "ERROR: STOKE not supported on your device. Please update flag field."
```

Comment:
Can you revert the second sentence back to the previous one.  I don't know what it means to "update flag field".

Reply:
ok